package.json

```diff
@@ -34,9 +34,7 @@
   "homepage": "https://github.com/readmeio/swagger-inline#readme",
   "dependencies": {
     "bluebird": "^3.4.1",
-    "chalk": "^3.0.0",
```

Comment:
Are we not using this anywhere? That surprises me! It can sometimes hide usage as it appends itself to the string prototype `"string".red`

Reply:
Yep, only place we were using it was in `src/cli.js` and I reworked the way we were displaying errors to throw actual errors so it's not needed anymore.